plugins/wm-actions/wm-actions.cpp

```diff
@@ -344,6 +340,21 @@ class wayfire_wm_actions_t : public wf::plugin_interface_t
             output->node_for_layer(wf::scene::layer::WORKSPACE),
             always_above);
 
+        auto wsize = output->workspace->get_workspace_grid_size();
+        showdesktop_active.resize(wsize.width);
+        for (int x = 0; x < wsize.width; x++)
+        {
+            showdesktop_active[x].resize(wsize.height);
```

Comment:
```suggestion
            showdesktop_active[x].resize(wsize.height);
            for (int y = 0; y < wsize.height; y++)
            {
                showdesktop_active[x][y] = false;
            }
```

and then omit the for loop below (on line 350)